solution/backend/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
```

Comment:
should we state `type: ClusterIP` to be more clear, even though it's the default?

Reply:
Added